packages/core/src/components/Form.tsx

```diff
@@ -62,6 +62,10 @@ export interface FormProps<T = any, F = any> {
    * to each field of the form. If you just want to disable some fields, see the `ui:disabled` parameter in `uiSchema`
    */
   disabled?: boolean;
+  /** When using `additionalProperties`, key collision is prevented by appending a unique integer to the duplicate key.
+   * This prop allows you to change the separator between the original key name and the integer. Default is "-"
+   */
+  duplicateKeySuffixSeparator?: string;
```

Comment:
Could we use the `idSeparator` instead? Less to pass around and has a similar use case

Reply:
Oh wait, update the `CHANGELOG.md` too...